src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/29.diff.sql

```diff
@@ -0,0 +1,236 @@
+/*************************************************************
+    TaskInfo table
+**************************************************************/
+
+IF NOT EXISTS (SELECT 'X' FROM SYS.COLUMNS WHERE OBJECT_ID = OBJECT_ID(N'TaskInfo') AND NAME = 'CreateDateTime')
+BEGIN
+ALTER TABLE dbo.TaskInfo
+ADD
+    CreateDateTime DATETIME2 (7) NOT NULL,
+    StartDateTime DATETIME2 (7) NULL,
+    EndDateTime DATETIME2 (7) NULL,
+    Worker varchar(100) NULL,
+    RestartInfo varchar(max) NULL,
+    CONSTRAINT DF_TaskInfo_CreateDate DEFAULT SYSUTCDATETIME() FOR CreateDateTime
+END
+
+GO
```

Comment:
Where is TaskInfo.sql change?

Reply:
Sorry for the miss, add change to TaskInfo.sql